agent_verification/src/connection/event.rs

```diff
@@ -0,0 +1,22 @@
+use cqrs_es::DomainEvent;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+pub enum ConnectionEvent {
+    SIOPv2AuthorizationResponseVerified { id_token: String },
+}
+
+impl DomainEvent for ConnectionEvent {
+    fn event_type(&self) -> String {
+        use ConnectionEvent::*;
+
+        let event_type: &str = match self {
+            SIOPv2AuthorizationResponseVerified { .. } => "AuthorizationRequestVerified",
```

Comment:
Shouldn't this be `"AuthorizationResponseVerified"`?

Reply:
Yeah indeed, I keep forgetting to change the values to be returned here when I changed the enum variant names. 

There's an easy fix for that that I will implement later, (issue #38)